apps/expo/src/components/AnnouncementTile.tsx

```diff
@@ -0,0 +1,55 @@
+import { Image, Text, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
+
+interface Props {
+  title: string;
+  titleImageSrc: ImageSourcePropType;
+  createdAt: Date;
+  createdBy: string;
+  userImageSrc: ImageSourcePropType;
+}
+
+function AnouncementTile({
+  title,
+  titleImageSrc,
+  createdAt,
+  createdBy,
+  userImageSrc,
```

Comment:
For now, you can use placeholders for the image sources. They will be added to announcements in a future sprint.